app/src/main/java/ch/admin/bag/dp3t/util/UserUploadInfoExtensions.kt

```diff
@@ -11,6 +11,6 @@ fun UserUploadInfo.toUploadVenueInfo(): UploadVenueInfo {
 		.setNotificationKey(ByteString.copyFrom(notificationKey))
 		.setIntervalStartMs(intervalStartMs)
 		.setIntervalEndMs(intervalEndMs)
-		.setFake(false)
+		.setFake(ByteString.copyFrom(ByteArray(1) { 0 }))
```

Comment:
Speaking of interfaces: why not let `setFake` still take a boolean, and then do the conversion internally?

Reply:
The setFake method is autogenerated from protobuf file and therefore can't be changed.